src/emails/UserCreatedNotification.tsx

```diff
@@ -0,0 +1,37 @@
+import EmailContent from "types/EmailContent"
+import UserDetails from "types/UserDetails"
+import { UserGroupResult } from "types/UserGroup"
+
+interface Props {
+  user: UserDetails
+}
+
+function printGroups(groups: UserGroupResult[]): string {
+  let groupString = ""
+
+  groups.forEach((group) => {
+    groupString += `${group.friendly_name}, `
```

Comment:
why not do a groups.map((x) => x.friendly_name).join(", ")

Reply:
you do not need the remove trailing commas afterwards

Reply:
I know which is why I have removed it with the regex below, that being said you way is more readable so I will adopt this approach.

Reply:
I will add `trimEnd` to the end though

Reply:
Review comment 2